src/PointwiseFunctions/AnalyticSolutions/NewtonianEuler/IsentropicVortex.cpp

```diff
@@ -3,118 +3,172 @@
 
 #include "PointwiseFunctions/AnalyticSolutions/NewtonianEuler/IsentropicVortex.hpp"
 
-#include <algorithm>
 #include <cmath>
 #include <cstddef>
-#include <ostream>
+#include <pup.h>  // IWYU pragma: keep
 
 #include "DataStructures/DataVector.hpp"                   // IWYU pragma: keep
 #include "DataStructures/Tensor/EagerMath/DotProduct.hpp"  // IWYU pragma: keep
 #include "DataStructures/Tensor/Tensor.hpp"
 #include "ErrorHandling/Assert.hpp"
-#include "Parallel/PupStlCpp11.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/PolytropicFluid.hpp"  // IWYU pragma: keep
+#include "Utilities/ConstantExpressions.hpp"
 #include "Utilities/GenerateInstantiations.hpp"
 #include "Utilities/Gsl.hpp"
 #include "Utilities/MakeWithValue.hpp"
 
+// We don't do anything complex, but IWYU thinks it's needed for some math
+// functions
+// IWYU pragma: no_include <complex>
+
 /// \cond
 namespace NewtonianEuler {
 namespace Solutions {
 
-IsentropicVortex::IsentropicVortex(
-    const double adiabatic_index, IsentropicVortex::Center::type center,
-    IsentropicVortex::MeanVelocity::type mean_velocity,
+template <size_t Dim>
+IsentropicVortex<Dim>::IsentropicVortex(
+    const double adiabatic_index, const std::array<double, Dim> center,
+    const std::array<double, Dim> mean_velocity,
```

Comment:
since you write out the explicit types instead of `typename Tag::type` (I prefer what you're doing here), just remove the `std::move`s and the `clang-tidy` comments